package/kedro_viz/models/experiment_tracking.py

```diff
@@ -111,3 +114,9 @@ def load_tracking_data(self, run_id: str):
 
 def get_dataset_type(dataset: AbstractVersionedDataSet) -> str:
```

Comment:
```suggestion
def get_full_dataset_type(dataset: AbstractVersionedDataSet) -> str:
    """e.g. kedro.extras.datasets.plotly.plotly_dataset.PlotlyDataSet or kedro_datasets.plotly.plotly_dataset.PlotlyDataSet"""
```
Maybe? Sort of depends how `dataset_type` is used elsewhere as to whether it's worth doing.

Reply:
we no longer have this. as get_dataset_module_class is the new 'get_dataset_type' e.g. pandas.csv_dataset.CSVDataset